commands/addtrain.js

```diff
@@ -12,14 +13,23 @@ module.exports = {
                 .setDescription(optdesc)
                 .setRequired(true)),
 	async execute(interaction) {
+        interaction.deferReply({ ephemeral: true }); // Sometimes it takes a sec to reply
+
+        if (!(await isOwner(interaction.member))) {
+            return interaction.followUp({content:'You don\'t have permission to do this.', ephemeral: true});
+        }
+
         const { options } = interaction
         if (config.trains.includes(options.getString(opt))) {
-            return interaction.reply({content: options.getString(opt) + ' is already being tracked!', ephemeral: true});
+            return interaction.followUp({content: `${options.getString(opt)} is already being tracked!`, ephemeral: true});
```

Comment:
I presume the change from reply -> follow up in this case is to account for latency/the reply delay, as opposed to adding on to an existing reply?